monailabel/utils/others/generic.py

```diff
@@ -307,6 +319,7 @@ def get_bundle_models(app_dir, conf, conf_key="models"):
                 download(name=name, version=version, bundle_dir=model_dir, source=zoo_source, repo=zoo_repo)
                 if version:
                     shutil.move(os.path.join(model_dir, name), p)
+        sys.path.append(p)
```

Comment:
why this change?  is your base version little outdated one?

Reply:
I see.. I think it did get removed.